johnsonandjohnson/Bodiless-JS
# Docs/naverror - fix build error

## Changes
Clean up documentation to remove warnings

## Test Instructions

- run npm build:doc and confirm no errors 
- run npm start and confirm no errors when visiting nagivation pages (or childern pages) or youtube pages.
- verify all navigation & children pages come up in docs as well as youtube page.
